pkg/types/baremetal/platform.go

```diff
@@ -33,10 +24,18 @@ type Platform struct {
 	// Default is qemu:///system
 	LibvirtURI string `json:"libvirtURI,omitempty"`
 
-	// IronicURI is the identifier for the Ironic connection.  It must be
-	// reachable from the host where the installer is run.
+	// ClusterProvisioningIP is the IP on the dedicated provisioning network
+	// where the baremetal-operator pod runs provisioning services,
+	// and http server to cache some downloaded content e.g RHCOS/IPA images
+	// These are used during the deployment of worker machines
```

Comment:
nit, this is used for Machine objects. in case of DR.. even master machines should be recreate able.. so I think we should drop this from the API.

Reply:
Ack yes it could also be used to rebuild master machines, will update